example/src/AndroidApp.tsx

```diff
@@ -0,0 +1,205 @@
+          method: 'POST',
+          headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            integrityToken,
+          }),
+        }
+      );
+      const response = await result.json();
+      setIsVerifyintegrityTokenLoading(false);
+      setDebugLog(JSON.stringify(response));
+    } catch (e) {
+      setIsVerifyintegrityTokenLoading(false);
+      setDebugLog(`${e}`);
+    }
+  };
+
+  const verifyAttestation = async () => {
+    try {
+      if (attestation === undefined) {
+        setDebugLog(
+          'Attestation is undefined, please call get attestation first.'
+        );
+        return;
+      }
+      setIsVerifyAttestationLoading(true);
+      const result = await fetch(
+        `${BACKEND_ADDRESS}/android/verifyAttestation`,
+        {
+          method: 'POST',
+          headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            attestation,
+          }),
+        }
+      );
+      const response = await result.json();
+      setIsVerifyAttestationLoading(false);
+      setDebugLog(JSON.stringify(response));
+    } catch (e) {
+      setIsVerifyAttestationLoading(false);
+      setDebugLog(`${e}`);
+    }
+  };
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <Text style={styles.h1}>Integrity Check Demo App</Text>
+      {isServiceAvailable ? (
+        <>
+          <Text>Play Integrity Standard Request</Text>
+          <ButtonWithLoader
+            title="Prepare"
+            onPress={() => prepare()}
+            loading={isPrepareLoading}
+          />
+          <ButtonWithLoader
+            title="Get token"
+            onPress={() => requestToken()}
+            loading={isRequestTokenLoading}
+          />
+          <ButtonWithLoader
+            title="Verify Token"
+            onPress={() => verifyToken()}
+            loading={isVerifyintegrityTokenLoading}
+          />
+          <Text>Key Attestation</Text>
+          <ButtonWithLoader
+            title="Get attestation"
+            onPress={() => requestAttestation()}
+            loading={isGetAttestationLoading}
+          />
+          <ButtonWithLoader
+            title="Verify attestation"
+            onPress={() => verifyAttestation()}
+            loading={isVerifyAttestationLoading}
+          />
+        </>
+      ) : null}
+      <ScrollView style={styles.debug}>
+        <Text>{debugLog}</Text>
+      </ScrollView>
+    </SafeAreaView>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+  },
+  h1: {
+    fontWeight: 'bold',
+    fontSize: 32,
+    textAlign: 'center',
+    marginTop: 50,
+    marginBottom: 50,
+  },
+  debug: {
+    width: '100%',
+    height: 300,
+    position: 'absolute',
+    bottom: 0,
+    backgroundColor: '#eaeaea',
+  },
+});
```

Comment:
```suggestion
return (
    <SafeAreaView style={styles.container}>
      <ScrollView contentContainerStyle={styles.scrollViewContentContainer}>
        <Text style={styles.h1}>Integrity Check Demo App</Text>
        {isServiceAvailable ? (
          <>
            <Text style={styles.h2}>Play Integrity Standard Request</Text>
            <ButtonWithLoader
              title="Prepare"
              onPress={() => prepare()}
              loading={isPrepareLoading}
            />
            <View style={styles.spacer} />
            <ButtonWithLoader
              title="Get token"
              onPress={() => requestToken()}
              loading={isRequestTokenLoading}
            />
            <View style={styles.spacer} />
            <ButtonWithLoader
              title="Verify Token"
              onPress={() => verifyToken()}
              loading={isVerifyintegrityTokenLoading}
            />
            <View style={styles.spacer} />
            <Text style={styles.h2}>Key Attestation</Text>
            <ButtonWithLoader
              title="Get attestation"
              onPress={() => requestAttestation()}
              loading={isGetAttestationLoading}
            />
            <View style={styles.spacer} />
            <ButtonWithLoader
              title="Verify attestation"
              onPress={() => verifyAttestation()}
              loading={isVerifyAttestationLoading}
            />
            <View style={styles.spacer} />
          </>
        ) : null}
        <>
          <Text>{debugLog}</Text>
        </>
      </ScrollView>
    </SafeAreaView>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
  },
  scrollViewContentContainer: {
    justifyContent: 'center',
    alignItems: 'center',
    flexGrow: 1,
    padding: 10,
  },
  h1: {
    fontWeight: 'bold',
    fontSize: 32,
    textAlign: 'center',
    marginTop: 16,
    marginBottom: 16,
  },
  h2: {
    fontWeight: 'bold',
    fontSize: 16,
    textAlign: 'center',
    marginTop: 16,
    marginBottom: 16,
  },
  spacer: {
    height: 8,
  },
  debug: {
    width: '100%',
    height: 300,
    position: 'absolute',
    bottom: 0,
    backgroundColor: '#eaeaea',
  },
});
```

Even if is a sample App, what about be more stylish 😀. I had some trouble on small display devices.

Reply:
Applied, thanks.